tests/test_index.cpp

```diff
@@ -63,7 +63,7 @@ TestIndex::FastGeneralTest(const std::string& name,
     // Test KnnSearch and RangeSearch
     {
         TestKnnSearch(index, dataset, search_parameters, top_k, 0.99);
-        TestRangeSearch(index, dataset, search_parameters, range, 0.99);
+        //        TestRangeSearch(index, dataset, search_parameters, range, 0.99);
```

Comment:
Why not run this test ?

Reply:
the range implement of the hgraph haven't been approve in PR #163